include/circt/Dialect/FIRRTL/FIRRTLDialect.h

```diff
@@ -22,9 +22,9 @@ namespace firrtl {
 
 class FIRRTLType;
 
-/// If the specified attribute list has a firrtl.name attribute, return its
+/// If the specified module has an argnames attribute, return its
```

Comment:
Nit: modules will always have this, so please make this more direct, e.g. "Return the argNames attribute for the specified module, which contains the name for each port."

I'd also recommend renaming this to `getModulePortNames` since it is in the firrtl namespace.  This also aligns with the `getModulePortName` function naming.